lib.nix

```diff
@@ -1,5 +1,18 @@
 { system ? builtins.currentSystem, config ? {} }:
 let
+  configWithOverrides = config // {
```

Comment:
On reflection, it would be good to have the whole definition of `config` in one place. It's in a slightly awkward place in `default.nix`, so I suggest doing what we do with the unfree predicate: define `packageOverrides` here, and then use it from `default.nix` and put it into `config` where it's defined.

Reply:
That means we'd have these overrides everywhere, but that seems fine.